level.go

```diff
@@ -14,6 +14,16 @@ const (
 	FATAL
 )
 
+const (
+	COLOUR_RESET  = "\033[0m"
+	COLOUR_RED    = "\033[31m"
+	COLOUR_REDBG  = "\033[41m"
+	COLOUR_GREEN  = "\033[32m"
+	COLOUR_YELLOW = "\033[33m"
+	COLOUR_BLUE   = "\033[34m"
+	COLOUR_WHITE  = "\033[97m"
+)
+
```

Comment:
Those could be configurable as well, but since it is the first iteration of this feature let's keep things easy :)

Reply:
Yep, I could add all other colour definitions already as well if you think it's neccessary, I've simply added the ones that I was going to use.